README.md

```diff
@@ -48,6 +48,52 @@ make package # Use -j to use more parallelism.
 
 Keep in mind that even though you've compiled the HTTP API with support for GenQuery2, that is half the story. You must also install the GenQuery2 package on the iRODS server which the HTTP API will connect to and the iRODS Catalog Service Provider.
 
+## Docker
+
+This project provides two Dockerfiles, one for building and one for running the application. GenQuery2 is enabled by default. As mentioned in the previous section, the iRODS server must have GenQuery2 installed before attempting to use the parser.
+
+### The Builder Image
+
+The builder image is responsible for building the iRODS HTTP API package. Before you can use it, you must build the image. To do that, run the following:
+```bash
+docker build -t irods-http-api-builder -f irods_builder.Dockerfile .
+```
+
+With the image built, you can now build the package by running the following:
+```bash
+docker run -it -v $PWD:/packages_output irods-http-api-builder
+```
+
+If everything succeeds, you will have a DEB package setting in the root of the repository.
+
+### The Runner Image
+
+The runner image is responsible for running the iRODS HTTP API. Building the runner image requires the iRODS HTTP API DEB package to exist in the root of the repository. See the previous section for details on generating the package.
+
+To build the image, run the following:
+```bash
+docker build -t irods-http-api-runner -f irods_runner.Dockerfile .
+```
+
+If all goes well, you will have a containerized iRODS HTTP API server! You can verify this by checking the version information. Below is an example.
+```bash
+docker run -it irods-http-api-runner -v
```

Comment:
Might be helpful to include the '$' here to differentiate the command from the output

Reply:
Done.